### PR TITLE
Add section on WebGPU

### PR DESCRIPTION
**Pull request by ccameron-chromium**
Indicate how color space is to be specified via GPUSwapChainDescriptor.

### CODE REVIEW

**Review by jdashg — APPROVED** [review]
This seems fine to me.

File: CanvasColorSpaceProposal.md [hunk 1]
```diff
@@ -157,6 +157,17 @@ function encode(val) {
...
+#### WebGPU behavior
+
+WebGPU's context configuration is specified dynamically using the `GPUCanvasContext` method `configureSwapChain`, which takes a `GPUSwapChainDescriptor` argument.
+Add an additional entry to `GPUSwapChainDescriptor` for the color space of the swap chain.
+<pre>
+partial dictionary GPUSwapChainDescriptor {
+  CanvasColorSpaceEnum colorSpace = "srgb";
+};
+</pre>
+Note that `GPUSwapChainDescriptor` already has a `texture` entry of type `GPUTextureFormat`.
```
Comment (kainino0x) [inline CanvasColorSpaceProposal.md:169]:
```suggestion
Note that `GPUSwapChainDescriptor` already has a `format` entry of type `GPUTextureFormat`.
```

Comment (kainino0x) [inline CanvasColorSpaceProposal.md:169]: Probably worth explicitly pointing out that the `GPUTextureFormat` also defines the encoding (srgb vs non-srgb), if that's right.

**Review by kenrussell — APPROVED** [review]
Looks good once @kainino0x 's review comments are addressed.

File: CanvasColorSpaceProposal.md (hunk 1 above)
Comment (ccameron-chromium) [reply CanvasColorSpaceProposal.md]: Oh, wait, GPUSwapChainDescriptor's IDL (that's checked in) has the entry `format`, not `texture` -- has that been changed?

Comment (kenrussell) [reply CanvasColorSpaceProposal.md]: Reference: https://gpuweb.github.io/gpuweb/#dictdef-gpuswapchaindescriptor

**Comment by ccameron-chromium** [comment]
Updated to rebase against the recent changes. Also added commentary on the interpretation of sRGB textures in WebGPU.

**Pull request merged**